### PR TITLE
docs: update payment docs for global price

### PR DESCRIPTION
**Pull request by forcodedancing**
## Description

Update docs to use `global` storage prices.

## Rationale

To align the new changes.

### CODE REVIEW

**Review by KeefeL — APPROVED** [review]

File: docs/guide/greenfield-blockchain/modules/billing-and-payment.md [hunk 1]
```diff
@@ -296,29 +297,35 @@ is resumed.
...
+- Secondary SP Store Price,
 - Primary SP Read Price;
-- Secondary SP Store Price.
 
-Every SP can set their own store price and read price via on-chain transactions.
-While the secondary SP store price is calculated as the median all SPs' store price.
+Every SP can set their own suggested store price and read price via on-chain transactions.
+At the first block of each month, the median all SPs' store prices will be calculated as 
+the Primary SP Store Price, the Secondary SP Store Price will be calculated as a proportion 
+of the Primary SP Store Price (e.g. 12%), and the median of all SPs' read prices will be
```
Comment (KeefeL) [inline docs/guide/greenfield-blockchain/modules/billing-and-payment.md:316]: Does the proportion can be governed?

File: docs/guide/greenfield-blockchain/modules/storage-provider.md [hunk 2]
```diff
@@ -120,7 +120,11 @@ message Params {
     (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
     (gogoproto.nullable) = false
   ];
+<<<<<<< HEAD
```
Comment (sysvm) [inline docs/guide/greenfield-blockchain/modules/storage-provider.md:123]: Please fix this.

File: docs/guide/greenfield-blockchain/modules/storage-provider.md [hunk 3]
```diff
@@ -120,7 +120,11 @@ message Params {
...
   // the ratio of the store price of the secondary sp to the primary sp, the default value is 80%
+=======
```
Comment (sysvm) [inline docs/guide/greenfield-blockchain/modules/storage-provider.md:125]: ditto

File: docs/guide/greenfield-blockchain/modules/storage-provider.md [hunk 4]
```diff
@@ -120,7 +120,11 @@ message Params {
...
+  // the ratio of the store price of the secondary sp to the primary sp, the default value is 12%
+>>>>>>> docs: update payment docs for global price
```
Comment (sysvm) [inline docs/guide/greenfield-blockchain/modules/storage-provider.md:127]: ditto

Comment (forcodedancing) [reply docs/guide/greenfield-blockchain/modules/storage-provider.md]: fixed

File: docs/guide/greenfield-blockchain/modules/storage-provider.md (hunk 3 above)
Comment (forcodedancing) [reply docs/guide/greenfield-blockchain/modules/storage-provider.md]: fixed

File: docs/guide/greenfield-blockchain/modules/storage-provider.md (hunk 2 above)
Comment (forcodedancing) [reply docs/guide/greenfield-blockchain/modules/storage-provider.md]: fixed

File: docs/guide/greenfield-blockchain/modules/billing-and-payment.md [hunk 5]
```diff
@@ -296,29 +297,35 @@ is resumed.
...
 - The primary SP will store the whole object file;
 - The secondary SPs will store part of the object file as a replica;
 - The primary SP will supply all the download requests of the object.
 
-There are 3 different on-chain prices:
+Hence, there are 3 different on-chain prices globally, which are used to charge storage 
+and download:
 
 - Primary SP Store Price;
+- Secondary SP Store Price,
```
[the remark this answers is not in the document]
Comment (forcodedancing) [reply docs/guide/greenfield-blockchain/modules/billing-and-payment.md]: reverted

File: docs/guide/greenfield-blockchain/modules/billing-and-payment.md (hunk 1 above)
Comment (forcodedancing) [reply docs/guide/greenfield-blockchain/modules/billing-and-payment.md]: Yes, added a sentence.

**Review by sysvm — APPROVED** [review]
LGTM

**Pull request merged**